spec/features/culqi_checkout_autocapture_spec.rb

```diff
@@ -0,0 +1,111 @@
+require "spec_helper"
+
+RSpec.describe "Culqi checkout autocapture", type: :feature do
+  before do
+    FactoryGirl.create(:store)
```

Comment:
I was planning to move this spec initialization into `spec_helper` in order to be more easy to reuse. But not sure will be the best place. Or maybe should create a file inside `spec/helpers` folder?

What do you think?